client.ps1

```diff
@@ -6,6 +6,7 @@ $ErrorActionPreference = "Stop"
 
 function Configure-TimeSync {
     Write-Output "Configuring Client VM to synchronize time with Studio VM..."
+    Start-Service w32time

```

Comment:
Use `Start-Service -Name w32time`.
According to docs, the default argument is a DisplayName but w32time is a name.